cluster/gce/util.sh

```diff
@@ -967,10 +973,18 @@ EOF
     fi
   fi
   if [[ "${ENABLE_CLUSTER_AUTOSCALER}" == "true" ]]; then
+      # TODO(mtaufen): AUTOSCALER_ENV_VARS is a hotfix for cluster autoscaler,
+      # which reads the kube-env to determine the shape of a node and was broken by #60020.
+      # This should be removed as soon as a more reliable source of information is available!
+      local node_labels=$(build-node-labels ${master})
```

Comment:
We should only need to do this on nodes (non-masters).

Reply:
done